### PR TITLE
[EEF-242] New rules for the template getting was added

### PR DESCRIPTION
**Pull request by Inferato**
**Description:** New rules for the template getting was added

**Youtrack:** https://youtrack.raccoongang.com/issue/EEF-242

**AC:**

1. A custom certificate template with organization set, returns as a default certificate for all courses in this organization.
2. A custom certificate template with course_id set, overrides default certificate template set for organization

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

### CODE REVIEW

File: common/djangoapps/util/organizations_helpers.py [hunk 1]
```diff
@@ -103,3 +104,10 @@ def organizations_enabled():
     Returns boolean indication if organizations app is enabled on not.
     """
     return settings.FEATURES.get('ORGANIZATIONS_APP', False)
+
+
+def get_course_organization(course_key):
```
Comment (GlugovGrGlib) [inline common/djangoapps/util/organizations_helpers.py:109]: Please name this function get_course_organization_name and add a docstring to this function

Comment (Inferato) [reply common/djangoapps/util/organizations_helpers.py]: Fixed

**Review by dyudyunov — COMMENTED** [review]
where is the migration file with a new model field?

File: lms/djangoapps/certificates/models.py [hunk 2]
```diff
@@ -1036,14 +1036,20 @@ class CertificateTemplate(TimeStampedModel):
...
-        help_text=_(u'Organization of template.'),
+        help_text=_(u'Organization id of template.'),
     )
     course_key = CourseKeyField(
         max_length=255,
         null=True,
         blank=True,
         db_index=True,
     )
+    organization = models.CharField(
```
Comment (dyudyunov) [inline lms/djangoapps/certificates/models.py:1047]: maybe we should name it `organization_name` too?

File: lms/djangoapps/certificates/models.py [hunk 3]
```diff
@@ -1036,14 +1036,20 @@ class CertificateTemplate(TimeStampedModel):
...
+        max_length=255,
+        null=True,
+        blank=True,
+        help_text=_(u'Organization name of template.Used if organization id is not set.')
```
Comment (GlugovGrGlib) [inline lms/djangoapps/certificates/models.py:1051]: Please fix a space here template.Used

**Review by GlugovGrGlib — CHANGES_REQUESTED** [review]
And you need the migration in the PR

**Review by GlugovGrGlib — APPROVED** [review]

**Comment by dyudyunov** [comment]
@Inferato you should squash commits before merging in the future. JFYI
Also, this MR includes excessive migrations (as for me)

**Comment by GlugovGrGlib** [comment]
Wow, missed it before merging. We were talking about force push, but now I can see the changes we talked about weren't applied.

**Comment by GlugovGrGlib** [comment]
In the future we surely must do so

**Pull request merged**